packages/polaris-viz/src/components/DonutChart/components/LegendValues/LegendValues.tsx

```diff
@@ -0,0 +1,95 @@
+import type {ColorVisionInteractionMethods, DataSeries} from 'index';
+import type {LabelFormatter} from '@shopify/polaris-viz-core';
+import {useTheme} from '@shopify/polaris-viz-core';
+
+import {TrendIndicator} from '../../../TrendIndicator';
+import {getTrendIndicatorData} from '../../../../utilities/getTrendIndicatorData';
+import {SquareColorPreview} from '../../../../components/SquareColorPreview';
+
+import styles from './LegendValues.scss';
+
+interface LegendContentProps {
+  data: DataSeries[];
+  labelFormatter: LabelFormatter;
+  getColorVisionStyles: ColorVisionInteractionMethods['getColorVisionStyles'];
+  getColorVisionEventAttrs: ColorVisionInteractionMethods['getColorVisionEventAttrs'];
+}
+
+export function LegendValues({
+  data,
+  labelFormatter,
+  getColorVisionStyles,
+  getColorVisionEventAttrs,
+}: LegendContentProps) {
+  const selectedTheme = useTheme();
+
+  const maxTrendIndicatorWidth = data.reduce((maxWidth, {metadata}) => {
+    if (!metadata?.trend) {
+      return maxWidth;
+    }
+
+    const {trendIndicatorWidth} = getTrendIndicatorData(metadata.trend);
+
+    return Math.max(maxWidth, trendIndicatorWidth);
+  }, 0);
+
+  return (
+    <table className={styles.Table}>
+      {data.map(({name, data, metadata}, index) => {
+        const value = data[0].value || 0;
```

Comment:
Is it valid for us to default the absence of data as 0 🤔 

I think its probably fine in the case of the donut chart but wanted to throw it out there that I don't think that that assumption is always valid. I wonder if we might be better off rendering something like `-` or omitting it completely if we don't have information about it?

Reply:
Yeah thats an interesting callout. Looking at other charts in Polaris viz, it seems when a data point has a value of null, no value is shown.

<img width="289" alt="Screenshot 2023-12-11 at 8 58 12 PM" src="https://github.com/Shopify/polaris-viz/assets/90475364/aa5ff7e3-ac22-4585-9681-a6f97399d324">

I think we should follow this pattern 👍 I'll check in with UX about if we want to omit the entire data point from the legend / maybe we just show nothing in the value cell.